frontend/pages/ManageControlsPage/Variables/Variables.tsx

```
@@ -11,7 +11,7 @@ import { IVariable } from "interfaces/variables";
 import { AppContext } from "context/app";
 
 import { stringToClipboard } from "utilities/copy_text";
-import { FLEET_WEBSITE_URL } from "utilities/constants";
+import { DEFAULT_USE_QUERY_OPTIONS, FLEET_WEBSITE_URL } from "utilities/constants";
```

ESLint

[error] Replace `·DEFAULT_USE_QUERY_OPTIONS,·FLEET_WEBSITE_URL·` with `⏎··DEFAULT_USE_QUERY_OPTIONS,⏎··FLEET_WEBSITE_URL,⏎`

(prettier/prettier)